src/HPRT/templates/dsmv_new.html

```diff
@@ -29,14 +29,32 @@ <h1> New DSM-V </h1>
 {% endif %}
 
 	{% csrf_token %}
+        <style>
```

Comment:
It looks like this same css is used for all of the different forms.  Maybe you could make a css file and then use the {% include %} importing to add it to the relevant html files?